package.json

```diff
@@ -13,10 +15,14 @@
         "@testing-library/react": "^12.1.2",
         "@testing-library/user-event": "^13.5.0",
         "axios": "^0.25.0",
+        "framer-motion": "^5.6.0",
```

Comment:
question: why did you install `framer-motion`? And also `mathjs`? And `react-colors`? :stuck_out_tongue: